clang/www/cxx_dr_status.html

```diff
@@ -2871,7 +2871,7 @@ <h2 id="cxxdr">C++ defect report implementation status</h2>
     <td><a href="https://cplusplus.github.io/CWG/issues/472.html">472</a></td>
     <td>drafting</td>
     <td>Casting across protected inheritance</td>
-    <td align="center">Not resolved</td>
+    <td class="none" align="center">No</td>
```

Comment:
For `"no drafting" status, can we say something different here? I think something like "Not resolved, probably no" would be better, given that we don't actually know what the resolution will be, and if it ends up resolved NAD then we actually do implement it correctly :-)

Reply:
Current state of things is my fault (I was the one who introduced `no open`, `no drafting`, and `no review` statuses). I've been pondering on a different idea recently: `No*`, and a pop-up saying something along the lines of `Tentative; issue hasn't been resolved yet`. Like cppreference does in their compiler support table. Seems less heavy for such a big table, but still provides details for those who are interested.

Another idea is for `No` to be a link to an issue on bug tracker instead of a pop-up.

It also worth mentioning that `make_cxx_dr_status` is strict with those unresolved statuses, and it yells every time status in a test doesn't match status in `cwg_index.html`.

Reply:
> For `"no drafting" status, can we say something different here? I think something like "Not resolved, probably no" would be better, given that we don't actually know what the resolution will be, and if it ends up resolved NAD then we actually do implement it correctly :-)

I think that would make sense. Any opinion @AaronBallman ?

Reply:
I don't think we should read tea leaves on unresolved issues; WG21 and WG14 will change direction on unresolved issues sometimes and so documenting anything about how we compare to the proposed resolution runs a reasonably high risk of getting stale. I think it's fine to have tests to document how we currently behave (and then if the test breaks, it's a reminder to whoever made the change to go look at the current status of the issue). But maybe we should just leave these as `Not Resolved` and make no other claims?

Otherwise, the information I think that's most accurate is whether Clang does or does not exhibit the issue that was reported (when possible). At least that tells the user "if you think this is an issue, Clang has that behavior."